doc/source/whatsnew/v0.16.0.txt

```diff
@@ -131,6 +131,37 @@ methods (:issue:`9088`).
     dtype: int64
 
 
+- During division involving a ``Series`` or ``DataFrame``, `0/0` and `0//0` now give `np.nan` instead of `np.inf`. (:issue:`9144`)
```

Comment:
@jreback, I tried summarizing this more as suggested.  Do you think it's fine now?